pyproject.toml

```diff
@@ -149,6 +172,20 @@ ignore = [
     # We use Black to format our code automatically, so we don't need PyLint to
     # check formatting for us.
     "E501", # line-too-long
+
+    "PLR2004", # Magic values, we mostly get it on HTTP status codes
+
+    # Disabled during the pylint migration, ideally we'll enable this after we are settled in ruff
```

Comment:
When you say "during the pylint migration", do you mean "while migrating away from pylint"?